platform/linux-generic/odp_crypto_openssl.c

```diff
@@ -2419,14 +2379,6 @@ odp_crypto_session_create(const odp_crypto_session_param_t *param,
 	case ODP_AUTH_ALG_AES_XCBC_MAC:
 		rc = process_aesxcbc_param(session, EVP_aes_128_ecb());
 		break;
-#if ODP_DEPRECATED_API
-	case ODP_AUTH_ALG_AES128_GCM:
-		if (param->cipher_alg == ODP_CIPHER_ALG_AES128_GCM)
-			aes_gcm = 1;
```

Comment:
You can actually remove this variable, aes_gcm. The way I understand, it was introduced to support deprecated ODP_AUTH_ALG_AES128_GCM when the switch case falls through

Reply:
Thanks, fixed in V2.